dotcom-rendering/src/palette.ts

```diff
@@ -711,6 +719,43 @@ const standfirstLinkBorderDark = (): string => {
 	return sourcePalette.neutral[46];
 };
 
+const standfirstBackground = ({
+	design,
+	display,
+	theme,
+}: ArticleFormat): string => {
+	switch (design) {
+		case ArticleDesign.LiveBlog:
+			switch (theme) {
+				case Pillar.Sport:
+					return sourcePalette.sport[100];
+				case ArticleSpecial.Labs:
+					return sourcePalette.news[200];
+				case ArticleSpecial.SpecialReport:
+					return sourcePalette.specialReport[300];
+				case ArticleSpecial.SpecialReportAlt:
+					return sourcePalette.news[200];
+				default:
+					return pillarPalette(theme, 200);
+			}
+		case ArticleDesign.DeadBlog:
+			switch (theme) {
+				case ArticleSpecial.SpecialReport:
+					return sourcePalette.specialReport[700];
+				default:
+					return sourcePalette.neutral[93];
+			}
+		default:
+			return articleBackgroundLight({ design, display, theme });
+	}
+};
+
+const standfirstBorder = ({ design }: ArticleFormat): string => {
+	if (design === ArticleDesign.LiveBlog) return 'rgba(255,255,255, 0.2)';
+	if (design === ArticleDesign.DeadBlog) return '#BDBDBD';
```

Comment:
Non-blocking

Again, looks like '#BDBDBD' isn't in source - @benwuersching or @HarryFischer, is this intentional?